objects/EncounterCards.40e930.json

```diff
@@ -71,7 +71,7 @@
     "posY": 1.551,
     "posZ": 47,
     "rotX": 0,
-    "rotY": 0,
+    "rotY": 180,
```

Comment:
What's this about?

Reply:
The first encounter deck was rotated incorrectly.

Reply:
That was intentional since the image looks better that way :D
I think the image might have the wrong rotation

Reply:
That’s….actually hilarious lmao. I’ll just change the original image when I fix the if statement thing from above.